src/submission-service.ts

```diff
@@ -246,11 +247,23 @@ async function processPendingQueue({
  * @param params
  * @returns
  */
-async function submit({
+async function submit(
+  params: SubmissionParams & {
+    autoSaveKey?: string
+  },
+): Promise<FormSubmissionResult> {
+  return await submitWithCompletionTimestamp({
+    ...params,
+    completionTimestamp: new Date().toISOString(),
+  })
+}
+
+async function submitWithCompletionTimestamp({
```

Comment:
Naming on this could probably be better, but the language part of my brain is winding down for the day right now.

Reply:
It is only an internal function to this file. 🤷